db/migrate/20210616103631_create_npq_profiles.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class CreateNpqProfiles < ActiveRecord::Migration[6.1]
+  def change
+    create_table :npq_profiles do |t|
+      t.references :user
+      t.references :npq_lead_provider
+      t.references :npq_course
+
+      t.date :date_of_birth
+      t.text :teacher_reference_number
+      t.boolean :teacher_reference_number_verified, default: false
+      t.text :school_urn
```

Comment:
I decided not to link up with actual `School` record. I don't feel it would give us any gain. Also I'm not sure if potentially ECF has a smaller dataset than NPQ??? We can link up via the `urn` at a later date if we need to

Reply:
We have the full dataset, including closed schools. Happy for this to stay as-is though